python/pypto/ir/compile.py

```
@@ -20,6 +22,38 @@
+    """
+    resolved_bin = shutil.which("ptoas")
+    if not resolved_bin:
+        raise FileNotFoundError(
+            "ptoas binary not found in PATH. Please install the ptoas package: pip install ptoas"
+        )
+
+    cmd = [resolved_bin, pto_path, "-o", output_path]
+    if ptoas_flags:
+        cmd.extend(ptoas_flags)
+
+    result = subprocess.run(cmd, capture_output=True, text=True, check=False)
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)